pages/linux/apx.md

```diff
@@ -1,17 +1,21 @@
 # apx
 
 > Package management utility for Vanilla OS.
-> Install packages inside managed containers from multiple sources (`apx` supports --aur,--dnf flags in all commands).
+> Install packages inside managed containers from multiple sources (`apx` supports --aur,--dnf, --apk flags in all commands).
 > More information: <https://github.com/Vanilla-OS/apx>.
 
-- Initialize the container:
+- Initialize/Reinitialize the container:
 
 `apx init`
```

Comment:
```suggestion
- Initialize or reinitialize the current container:

`apx init`
```

Optional fix: unexpected `/` delimiter, `or` expected.
Optional fix: unexpected `container` noun, `current` expected.

Reply:
`the current` won't be apt here, I will instead use a specific container as this command supports flags too.